CWP/papers/HSF-CWP-2017-04_doma/latex/authors.tex

```diff
@@ -2,36 +2,36 @@
 \author[16]{Riccardo Maria Bianchi,}
 \author[2]{Ian Bird,}
 \author[13]{Brian Bockelman,}
-\author[2]{Simone Campana,}
+\author[2,a]{Simone Campana,}
```

Comment:
This is seems wrong to me - surely in the DOMA paper the idea would be identify the editors of **this** paper, not to pick out the editors of the roadmap paper (I suppose you got this because you used the footnotes of the roadmap paper). So it should be Bo and you who are editors here.